test/e2e/config/ibmcloud-e2e.yaml

```diff
@@ -48,7 +48,25 @@ variables:
   # Cluster Addons
   CNI: "${PWD}/test/e2e/data/cni/calico/calico.yaml"
   IP_FAMILY: "IPv4"
+  # Following variables should be set based on the flavour being tested
```

Comment:
Note: cluster template management based on the different flavors will be taken care of in upcoming PRs to better manage these variables(exploring alternatives)

Reply:
@mkumatag , please hold this, I will make the changes for different templates and push in this PR itself. Its a dependency for the CI.